pkg/openstack/loadbalancer_test.go

```diff
@@ -534,3 +537,63 @@ func Test_getListenerProtocol(t *testing.T) {
 		})
 	}
 }
+
+func TestLbaasV2_GetLoadBalancerName(t *testing.T) {
+	lbaas := &LbaasV2{}
+	var ctx context.Context
+
+	type testArgs struct {
+		ctx         context.Context
+		clusterName string
+		service     *corev1.Service
+	}
+
+	testArg := &testArgs{
+		ctx:         ctx,
+		clusterName: "my_valid_cluster",
+		service: &corev1.Service{
+			ObjectMeta: v1.ObjectMeta{
+				Namespace: "valid_cluster_namespace",
+				Name:      "valid_name",
+			},
+		},
```

Comment:
It's a minor thing, but Kubernetes will not allow names containing "_": https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names